flight/Modules/RadioComBridge/RadioComBridge.c

```diff
@@ -136,31 +127,23 @@ static int32_t RadioComBridgeStart(void)
 		// Check if this is the coordinator modem
 		data->isCoordinator = PIOS_RFM22B_IsCoordinator(PIOS_COM_RFM22B);
```

Comment:
Since you're eliminating so many uses of `data`, could you refactor it's name to `radioComBridgeData`? I know it's a little wordy, but it just makes it so much easier to find in a project because there are fewer collisions with all the strings containing just `data`.